edx-platform/pearson-vue-theme/lms/student-account/login.underscore

```diff
@@ -1,80 +1,70 @@
-        <% } %>
-        <%- gettext("If you do not yet have an account, use the button below to register.") %>
+      <% if ( context.providers.length > 0 && !context.currentProvider || context.hasSecondaryProviders ) { %>
+        <%- gettext("Sign in here using your email address and password, or use one of the providers listed below.") %>
+      <% } else { %>
+        <%- gettext("Sign in here using your email address and password.") %>
+      <% } %>
+      <%- gettext("If you do not yet have an account, use the button below to register.") %>
     </p>
 
     <% if ( context.hasSecondaryProviders ) { %>
-        <button type="button" class="button-secondary-login form-toggle" data-type="institution_login">
-            <%- gettext("Use my university info") %>
-        </button>
+      <button type="button" class="button-secondary-login form-toggle" data-type="institution_login">
+        <%- gettext("Use my university info") %>
+      </button>
 
-        <div class="section-title lines">
-            <h3>
-                <span class="text"><%- gettext("or sign in with") %></span>
-            </h3>
-        </div>
-    <% } %>
+      <div class="section-title lines">
+        <h3>
+          <span class="text"><%- gettext("or sign in with") %></span>
+        </h3>
+      </div>
+      <% } %>
 
     <%= HtmlUtils.HTML(context.fields) %>
 
     <button type="submit" class="action action-primary action-update js-login login-button"><%- gettext("Sign in") %></button>
 
     <% if ( context.providers.length > 0 && !context.currentProvider) { %>
-    <div class="login-providers">
+      <div class="login-providers">
         <div class="section-title lines">
-            <h3>
-                <span class="text"><%- gettext("or sign in with") %></span>
-            </h3>
+          <h3>
+            <span class="text"><%- gettext("or sign in with") %></span>
+          </h3>
         </div>
 
-        <% _.each( context.providers, function( provider ) {
-            if ( provider.loginUrl ) { %>
-                <button type="button" class="button button-primary button-<%- provider.id %> login-provider login-<%- provider.id %>" data-provider-url="<%- provider.loginUrl %>">
-                    <div class="icon <% if ( provider.iconClass ) { %>fa <%- provider.iconClass %><% } %>" aria-hidden="true">
-                        <% if ( provider.iconImage ) { %>
-                            <img class="icon-image" src="<%- provider.iconImage %>" alt="<%- provider.name %> icon" />
-                        <% } %>
-                    </div>
-                    <span aria-hidden="true"><%- provider.name %></span>
-                    <span class="sr"><%- _.sprintf( gettext("Sign in with %(providerName)s"), {providerName: provider.name} ) %></span>
-                </button>
-            <% }
-        }); %>
-    </div>
+        <% _.each( context.providers, function( provider ) {1
```

Comment:
Could you remove the number 1 at the end of this line?. It seems not to have a purpose. @luismorenolopera

Reply:
ok